### ISSUE TITLE
Project Idea

### ISSUE THREAD

**Issue by alexlupo5** [AlexLupo5/CSC212-Term-Project#3]
Possible idea for project:

Disease/virus tracking:
Maps the locations of where disease/virus has spread using convex hull.
Can be used to show the spread over a map
Can also be used as a way to check if an area is safe from a disease/virus

**Comment by JakeN237** [comment]
Yes I think this is a great idea and we should use this for the project.  I think it is a great adaptation of Convex Hull being used in a real world setting.
